src/editing_tools/AdjustTool.vala

```diff
@@ -344,10 +343,7 @@ public class EditingTools.AdjustTool : EditingTool {
                 return photo.equals (owner.canvas.photo);
             }
 
-            AdjustEnhanceCommand enhance_command = (AdjustEnhanceCommand) command;
-            if (enhance_command == null) {
-                return false;
-            }
+            var enhance_command = (AdjustEnhanceCommand) command;
```

Comment:
Need safety check

Reply:
I see. I'll probably edit the others I see along the way

Reply:
```suggestion
            if (command == null || !(command is AdjustEnhanceCommand) {
                 return false;
            }
     
            var enhance_command = (AdjustEnhanceCommand) command;
```

There is another place above this where a similar check should be used for safety but it has not yet been edited so I cannot make a suggestion.